### PR TITLE
Term entry] sql math functions atan

### PR DESCRIPTION
**Pull request by dev-03960**
### Type of Change

- Adding a new entry atan.md
- Explaining the atan function

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/sql/concepts/math-functions/terms/atan/atan.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
```
Comment (SSwiniarski) [inline content/sql/concepts/math-functions/terms/atan/atan.md:1]: This is not a properly formatted metadata section. You can learn how to put together a metadata section in the style guide [here](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md#metadata).

File: content/sql/concepts/math-functions/terms/atan/atan.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+title: ATAN Function
+short_description: Explanation and usage of the ATAN function in SQL.
+---
+
+The ATAN function in SQL is a mathematical function used to calculate the arctangent of a given number. It returns the angle (in radians) whose tangent is the specified number. This function is useful in various mathematical and analytical calculations, such as determining angles or performing trigonometric operations in SQL queries.
```
Comment (SSwiniarski) [inline content/sql/concepts/math-functions/terms/atan/atan.md:6]:
```suggestion
The **`ATAN()`** function in SQL is a mathematical function used to calculate the arctangent of a given number. It returns the angle (in radians) whose tangent is the specified number. This function is useful in various mathematical and analytical calculations, such as determining angles or performing trigonometric operations in SQL queries.
```

File: content/sql/concepts/math-functions/terms/atan/atan.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+## SQL Database Compatibility
```
Comment (SSwiniarski) [inline content/sql/concepts/math-functions/terms/atan/atan.md:8]:
```suggestion
```
Let's remove this header and the blank line that precedes it.

File: content/sql/concepts/math-functions/terms/atan/atan.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+The syntax of the ATAN function in SQL is as follows:
+
+```sql
+ATAN(expression)
+```
+
+## Behavior
+
+- If the input expression is a numeric value, the ATAN function returns the arctangent of that value in radians.
```
Comment (SSwiniarski) [inline content/sql/concepts/math-functions/terms/atan/atan.md:22]:
```suggestion
- If the input expression is a numeric value, the `ATAN()` function returns the arctangent of that value in radians.
```

File: content/sql/concepts/math-functions/terms/atan/atan.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+- If the input expression is NULL, the ATAN function also returns NULL.
```
Comment (SSwiniarski) [inline content/sql/concepts/math-functions/terms/atan/atan.md:23]:
```suggestion
- If the input expression is `NULL`, the `ATAN()` function also returns `NULL`.
```

File: content/sql/concepts/math-functions/terms/atan/atan.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+
+## Examples
```
Comment (SSwiniarski) [inline content/sql/concepts/math-functions/terms/atan/atan.md:25]:
```suggestion
## Example
```

File: content/sql/concepts/math-functions/terms/atan/atan.md [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+
+1. Calculating the arctangent of a specific value:
```
Comment (SSwiniarski) [inline content/sql/concepts/math-functions/terms/atan/atan.md:27]:
```suggestion
This example calculates the arctangent of a specific value:
```

File: content/sql/concepts/math-functions/terms/atan/atan.md [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+
+```sql
+SELECT ATAN(1.5);
+
+SELECT id, value, ATAN(value) AS arctan_value
```
Comment (SSwiniarski) [inline content/sql/concepts/math-functions/terms/atan/atan.md:32]: Remove this second example and all the extra blank lines from this code block.

File: content/sql/concepts/math-functions/terms/atan/atan.md [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+FROM my_table;
+
+
+
+```
+
```
Comment (SSwiniarski) [inline content/sql/concepts/math-functions/terms/atan/atan.md:38]: Delete everything below here.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@dev-03960 I did a first review and left a few comments. You especially need to wok on the metadata. You also need to sign the [CLA](https://cla-assistant.io/Codecademy/docs?pullRequest=2419).

File: content/sql/concepts/math-functions/terms/atan/atan.md (hunk 1 above)
Comment (dev-03960) [reply content/sql/concepts/math-functions/terms/atan/atan.md:1]: ok sir I will do that thank you for reviewing

File: content/sql/concepts/math-functions/terms/atan/atan.md [hunk 10]
```diff
@@ -0,0 +1,35 @@
+---
```
Comment (SSwiniarski) [inline content/sql/concepts/math-functions/terms/atan/atan.md:1]: You still need to update the metadata to be properly formatted. The rules for the metadata can be found [here](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md#metadata).

File: content/sql/concepts/math-functions/terms/atan/atan.md [hunk 11]
```diff
@@ -0,0 +1,35 @@
...
+
+## Example
+
+This example calculates the arctangent of a specific value:
+
+```sql
+SELECT ATAN(1.5) AS Arctangent_Result;
+```
+
+In this SQL code example, the **`ATAN()`** function is used to compute the arctangent of the value 1.5. The result is given the alias "Arctangent_Result" using the AS keyword, which allows you to assign a name to the column representing the calculated value.
```
Comment (SSwiniarski) [inline content/sql/concepts/math-functions/terms/atan/atan.md:29]:
```suggestion
In this SQL code example, the **`ATAN()`** function is used to compute the arctangent of the value 1.5. The result is given the alias "Arctangent_Result" using the AS keyword, which assigns a name to the column representing the calculated value.
```

File: content/sql/concepts/math-functions/terms/atan/atan.md [hunk 12]
```diff
@@ -0,0 +1,35 @@
...
+
+The **`SELECT`** statement is used to retrieve the result of the calculation. By executing this query, the database system will evaluate the arctangent of 1.5 and return the result. The specific output may vary depending on the database system and its default unit for angles (radians or degrees).
+
+For instance, if you execute this code in MySQL, the output would be approximately 0.9828 if the output is in radians. If the output is in degrees, the result would be approximately 56.31 degrees. The exact output would depend on the database system and its configuration.
```
Comment (SSwiniarski) [inline content/sql/concepts/math-functions/terms/atan/atan.md:33]:
```suggestion
For instance, when executed in MySQL, the output would be approximately 0.9828 if the output is in radians. If the output is in degrees, the result would be approximately 56.31 degrees. The exact output would depend on the database system and its configuration.
```

File: content/sql/concepts/math-functions/terms/atan/atan.md [hunk 13]
```diff
@@ -0,0 +1,35 @@
...
+
+By using this code snippet as a basis, you can substitute different values inside the **`ATAN()`** function to calculate the arctangent of other numbers in your SQL queries. The resulting value can be retrieved with a specific alias to make the output more descriptive and meaningful.
```
Comment (SSwiniarski) [inline content/sql/concepts/math-functions/terms/atan/atan.md:35]: You can delete this line.

**Review by SSwiniarski — COMMENTED** [review]
@dev-03960 Some more comments, and you still need to fix the metadata.

File: content/sql/concepts/math-functions/terms/atan/atan.md [hunk 14]
```diff
@@ -0,0 +1,37 @@
+---
+Title: ATAN Function
```
Comment (SSwiniarski) [inline content/sql/concepts/math-functions/terms/atan/atan.md:2]:
```suggestion
Title: 'ATAN()'
```

File: content/sql/concepts/math-functions/terms/atan/atan.md [hunk 15]
```diff
@@ -0,0 +1,37 @@
...
+Description: Trigonometric function returning the arctangent angle in SQL
```
Comment (SSwiniarski) [inline content/sql/concepts/math-functions/terms/atan/atan.md:3]:
```suggestion
Description: 'Returns the arctangent of a provided number.'
```

File: content/sql/concepts/math-functions/terms/atan/atan.md [hunk 16]
```diff
@@ -0,0 +1,37 @@
...
+Subjects:
```
Comment (SSwiniarski) [inline content/sql/concepts/math-functions/terms/atan/atan.md:4]: The subjects need to be in single quotes and need to come from [this list](https://github.com/Codecademy/docs/blob/main/documentation/subjects.md).

File: content/sql/concepts/math-functions/terms/atan/atan.md [hunk 17]
```diff
@@ -0,0 +1,37 @@
...
+  - SQL
+  - Mathematics
+Tags:
```
Comment (SSwiniarski) [inline content/sql/concepts/math-functions/terms/atan/atan.md:7]: The tags need to be in single quotes and need to come from [this list](https://github.com/Codecademy/docs/blob/main/documentation/tags.md).

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@dev-03960 still need some updates to the metadata.

**Review by SSwiniarski — APPROVED** [review]
@dev-03960 I think this looks good for a second review.

**Comment by dev-03960** [comment]
hello sir I solve the Term entry] swift arrays .shuffle() (PR 2471) generate the pull request kindly review that also for successful merging

File: content/sql/concepts/math-functions/terms/atan/atan.md [hunk 18]
```diff
@@ -0,0 +1,40 @@
...
+  - 'MySQL'
+  - 'PostgreSQL'
+  - 'SQLite'
+  - 'Functions'
+CatalogContent:
+ - 'learn-sql'
+ - 'paths/analyze-data-with-sql'
+---
+
+The **`ATAN()`** function in SQL is a mathematical function utilized to compute the arctangent of a provided number. It evaluates the angle, measured in radians, whose tangent corresponds to the specified number. This function finds its utility in numerous mathematical and analytical calculations, such as determining angles or performing trigonometric operations within SQL queries.
```
Comment (caupolicandiaz) [inline content/sql/concepts/math-functions/terms/atan/atan.md:19]:
```suggestion
The **`ATAN()`** function in SQL is a mathematical function utilized to compute the arctangent of a provided number. The function takes an angle, measured in radians, and returns the arctangent, also known as the inverse of the tangent. This function is used in numerous mathematical and analytical calculations, such as determining angles or performing trigonometric operations within SQL queries.
```

File: content/sql/concepts/math-functions/terms/atan/atan.md [hunk 19]
```diff
@@ -0,0 +1,40 @@
...
+
+The **`ATAN()`** function is widely supported across various SQL database systems, including MySQL, PostgreSQL, Oracle, and Microsoft SQL Server. Its availability in these systems enables users to perform complex calculations involving angles and trigonometry directly within their SQL queries, providing a convenient and efficient means of processing numerical data. Whether it's calculating the inclination of lines, performing geometric transformations, or solving problems involving angles, the `ATAN()` function proves to be a valuable tool for mathematical computations within the SQL environment.
```
Comment (caupolicandiaz) [inline content/sql/concepts/math-functions/terms/atan/atan.md:21]:
```suggestion
> **Note:** `ATAN()` is widely supported across various SQL database systems, including MySQL, PostgreSQL, Oracle, and Microsoft SQL Server. Review the documentation of the specific database system for implementation details.
```

File: content/sql/concepts/math-functions/terms/atan/atan.md [hunk 20]
```diff
@@ -0,0 +1,40 @@
...
+
+## Syntax
+
+The syntax of the **`ATAN()`** function in SQL is as follows:
```
Comment (caupolicandiaz) [inline content/sql/concepts/math-functions/terms/atan/atan.md:25]:
```suggestion
```

File: content/sql/concepts/math-functions/terms/atan/atan.md [hunk 21]
```diff
@@ -0,0 +1,40 @@
...
+
+```sql
```
Comment (caupolicandiaz) [inline content/sql/concepts/math-functions/terms/atan/atan.md:27]:
```suggestion
```pseudo
```

File: content/sql/concepts/math-functions/terms/atan/atan.md [hunk 22]
```diff
@@ -0,0 +1,40 @@
...
+ATAN(expression)
+```
+
+- If the input expression is a numeric value, the **`ATAN()`** function returns the arctangent of that value in radians.
+- If the input expression is `NULL`, the **`ATAN()`** function also returns `NULL`.
```
[the remark this answers is not in the document]
Comment (caupolicandiaz) [reply content/sql/concepts/math-functions/terms/atan/atan.md:32]: The function should only be formatted in bold once at the beginning of the entry

File: content/sql/concepts/math-functions/terms/atan/atan.md [hunk 23]
```diff
@@ -0,0 +1,40 @@
...
+
+## Example
+
+This example calculates the arctangent of a specific value:
+
+```sql
```
Comment (caupolicandiaz) [inline content/sql/concepts/math-functions/terms/atan/atan.md:38]: There should be an additional output block showing the result of the example code

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@dev-03960 Please update the entry, and tag me when you've finished making changes. Thanks

**Review by caupolicandiaz — APPROVED** [review]
@dev-03960 Looks good. I did edit the output, which we usually include in a shell block or table and made a couple of spacing edits.

**Comment by dev-03960** [comment]
@caupolicandiaz now is it successfully merging this pr or other checks must be required

**Comment by dev-03960** [comment]
Please Visible my name in the contributor in atan() I follow the steps
which was given in this discuss forum
https://discuss.codecademy.com/t/my-contribution-is-live-but-im-not-being-shown-as-a-contributor-why-is-this/758036

**Comment by SSwiniarski** [comment]
@dev-03960 You need to give it a little time before the credit appears. It's there now.

image

**Comment by dev-03960** [comment]
yes sir I checked now it's there
thank you for your support and guidance

**Pull request merged**